### PR TITLE
Add snippets for `OR` queries

### PR DESCRIPTION
**Pull request by jhuleatt**
To be used in the docs here: https://firebase.google.com/docs/firestore/query-data/queries#or_queries

~can be merged once https://github.com/firebase/firebase-admin-node/pull/2192 is released~ it has been released!

### CODE REVIEW

File: firestore/main/package.json [hunk 1]
```diff
@@ -12,12 +12,13 @@
   "license": "ISC",
   "dependencies": {
     "@google-cloud/firestore": "^4.4.0",
```
Comment (thatfiredev) [inline firestore/main/package.json:14]: OR queries where introduced in [v6.5.0](https://github.com/googleapis/nodejs-firestore/releases/tag/v6.5.0). We'll need to bump this :)

Comment (jhuleatt) [reply firestore/main/package.json:14]: Huh, I wonder why that's in here as a dependency. Seems like it should actually be removed - I tried updating it but it didn't change anything, and the Firebase admin sdk already [specifies `6.5.0` as a dep](https://github.com/firebase/firebase-admin-node/blob/de026a1a329c077ed22b2c1a3ee2fa324b4fb497/package.json#L209) (though it is optional??)

Comment (jhuleatt) [reply firestore/main/package.json:14]: I filed an issue for the missing re-export: https://github.com/firebase/firebase-admin-node/issues/2191

Comment (jhuleatt) [reply firestore/main/package.json]: A new version of the Admin SDK has been released that fixes this! https://github.com/firebase/firebase-admin-node/releases/tag/v11.9.0

**Comment by jhuleatt** [comment]
@thatfiredev this is the main file to review: [firestore/main/index.js](https://github.com/firebase/snippets-node/pull/364/files#diff-e01a739ab2a8bd1334662bb095cf70d85751fb2ce7c06115132a62d6707e3a3e)

**Review by thatfiredev — APPROVED** [review]
LGTM

**Pull request merged**